src/hooks/usePager.js

```diff
@@ -0,0 +1,23 @@
+import { useContext, useCallback } from 'react'
+import { manifestContext } from '../components/Manifest'
+import * as pagerLogic from '../utils/pagerLogic'
+import * as actions from '../actions'
+
+export default () => {
+  const { state, dispatch } = useContext(manifestContext)
+  const { page, pageSize, count } = state
```

Comment:
after switching to useManifest, we shouldn't need all of these double unpacking lines